docs/guides/nginx-configuration.md

```diff
@@ -1,7 +1,9 @@
 ### Notes & Warnings
 
 - **This is an unsupported configuration created by the community**
+- If you're using Raspbian Buster (debian 10), then you should install 'php7.3-fpm' instead of v7.0, and change all instances of 'php7.0-fpm' to 'php7.3-fpm'.
```

Comment:
```suggestion
- If you're using Raspbian Buster (Debian 10), then you should install 'php7.3-fpm' instead of v7.0, and change all instances of 'php7.0-fpm' to 'php7.3-fpm'.
```

Reply:
Debian is a proper name.

Reply:
It should be `Debian` for consistency, but anyway, your call.

Reply:
Yes, it need to be `Debian`.